src/www/model/server.ts

```diff
@@ -40,10 +43,21 @@ export interface Server {
   checkReachable(): Promise<boolean>;
 }
 
-export type ServerConfig = ShadowsocksConfig;
+
+export interface ServerConfig {
```

Comment:
Let's remove ServerConfig from the model. That's a persistency/application concept.
For persistency, let's use {name, access_key}, This way we reuse the existing serialization format. No need to have two different formats.